ICA/auto_Granduer_ICA.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date,datetime
+
+# This funciton uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+# While loop is infinate will keep script running as long as it dosent error out.
+go=1
+while go == 1:
+    print('Looking for anaylses that need to start!')
+    #This block set the icav2 tool to correct project in use
+    bashCommand="icav2 projects enter Testing"
+    process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+    process.communicate()
+
+    #This  block collects the information of data and runs on ICA
+    bashCommand = "icav2 projectdata list --file-name UT-M03999-  --parent-folder / --match-mode FUZZY"
+    tmp= icav_out(bashCommand)
+    bashCommand = "icav2 projectdata list --file-name UT-M07101-  --parent-folder / --match-mode FUZZY"
+    tmp.extend(icav_out(bashCommand))
+
+    #This block is needed becuase we started to stream data to ICA before pipelines were working
+    cutoff=datetime.strptime('220709', '%y%m%d')
+    for i in range(len(tmp)-1,-1,-1):
+        try:
+            date=datetime.strptime(tmp[i].split()[1].split('-')[2].split('_')[0], '%y%m%d')
+        except:
+            del tmp[i]
+            continue
+        if date < cutoff:
```

Comment:
Aren't all the dates greater than the cutoff now?

Reply:
Yes, I have the script running and would like it to kick off just for new runs. No need to run Granduer twice.

Reply:
I think that's a good idea. Because it would start exploring all the old runs and take forever. 

What if the cutoff was something like 

```
from datetime import date, datetime, timedelta
delta = 10

today = datetime.now()    
recent = today - timedelta(days=delta)

cutoff=datetime.strptime(recent, '%y%m%d')
```

Reply:
I like this only needs slight modification. Recent is already a datetime object; so last line is unneeded.